src/main/java/org/folio/rest/core/RestClient.java

```diff
@@ -28,12 +30,11 @@ public <T> CompletableFuture<T> getById(String baseEndpoint, String id, RequestC
     return get(requestEntry, requestContext, responseType);
   }
```

Comment:
getById used only in test, lets delete it.
Basically We had refactoring previously to not use CompletableFuture from java.util.concurrent and use only Vertx future in order to not have mixing of concurrent architectures, seems this place was forgotten to refactor